content/guides/installing-python/_index.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
This page needs to be added to the top level guides/index.md page as well. It should go in the General section, ordered alphabetically with the rest. The icon can be the same one used for the `Installing board drivers` page.